Hearthstone/app/src/main/res/layout/fragment_card_overview.xml

```diff
@@ -0,0 +1,163 @@
+            android:gravity="center_horizontal"
+            android:textColor="@color/white"
+            android:textSize="@dimen/size_18"
+            android:textStyle="normal"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:text="Mana Bind" />
+
+        <ImageView
+            android:id="@+id/backToSearchPage"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/margin_16"
+            app:layout_constraintBottom_toBottomOf="@+id/cardName"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="@+id/cardName"
+            app:srcCompat="@drawable/arrow_chevron_big_left" />
+
+        <TextView
+            android:id="@+id/className"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/margin_9"
+            android:fontFamily="@font/montserrat_medium"
+            android:gravity="center_horizontal"
+            android:textColor="@color/yellow_letters"
+            android:textSize="@dimen/size_18"
+            android:textStyle="normal"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/cardName"
+            tools:text="Mage" />
+
+        <CheckBox
+            android:id="@+id/icon"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignParentTop="true"
+            android:layout_alignParentEnd="true"
+            android:layout_marginEnd="@dimen/margin_16"
+            android:button="@drawable/selector_favlist_item"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/className"
+            app:useMaterialThemeColors="false" />
+
+        <ImageView
+            android:id="@+id/cardImage"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/height_0"
+            android:layout_marginStart="@dimen/margin_50"
+            android:layout_marginEnd="@dimen/margin_50"
+            android:layout_marginBottom="@dimen/margin_20"
+            android:src="@drawable/photo_copy_2"
+            app:layout_constraintBottom_toTopOf="@+id/cardView"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/icon" />
+
+        <androidx.cardview.widget.CardView
+            android:id="@+id/cardView"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/margin_140"
```

Comment:
Why do you need so much bottom margin?

Reply:
I changed it to 100dp, but it's still too much since in zeplin this cardView is above the navigation bar, where it originally has 59dp margin. This will change when I add the navigation bar

Reply:
app:layout_constraintBottom_toBottomOf="@+id/particles_blue". This looks weird. You should be constraining to bottom of parent?

Reply:
Ahh sorry, what happens is that there is an image of stars (particles_blue) that covers the entire width and height, maybe that's why it was joined to that image and not the parent, sorry, I'll fix it